mangaki/mangaki/utils/als.py

```diff
@@ -23,6 +23,7 @@ def load(self, filename):
         self.M = backup.M
         self.U = backup.U
         self.VT = backup.VT
+        self.means = backup.means
```

Comment:
Because `means` are needed for `predict`. So whenever I was loading the model from a backup, I could not predict anymore ^^'